File: scriptlandia-tech-examples/aspect/09-testcase-generator/src/main/java/HelloWorld.java
public class HelloWorld {
        
  public void test1()  {
    System.out.println("Executing test1().");

    test2();
  }

  public void test2()  {
    System.out.println("Executing test2().");
  }

  public static void main (String[] args) {
    System.out.println("Hello World!");

    HelloWorld helloWorld = new HelloWorld();

    helloWorld.test1();
    helloWorld.test2();
  }

}

File: scriptlandia-tech-examples/other/swt/glimmer/hello_world.rb
# Copyright (C) 2007-2008 Annas Al Maleh
# Licensed under the LGPL. See /COPYING.LGPL for more details.

require "java"

jrubyHome = java.lang.System.getProperty("jruby.home")

require jrubyHome + "/lib/ruby/gems/1.8/gems/glimmer-0.1.0.0/src/swt"

class HelloWorld
	include_package 'org.eclipse.swt'
	include_package 'org.eclipse.swt.layout'
	
	include Glimmer

	def launch
		@shell = shell {
			text "SWT"
			composite {
				label { 
					text "Hello World!" 
				}
			}
		}

    @shell.open
	end
end

HelloWorld.new.launch
